src/store/zustand/store.ts

```diff
@@ -8,9 +8,11 @@ import create, { StoreApi, UseBoundStore } from 'zustand';
 export type AppState = {
 	calendarView: string;
 	date: Date;
+	isResumeOpen: boolean;
```

Comment:
i find the name quite unrelated to its function, shouldn't it be isPreviewOpen?

Reply:
it should but the component is name event-resume-view so in order to not make a developer confuse I kept that name